lib/tasks/ingest.rake

```diff
@@ -389,6 +389,44 @@ namespace :ingest do
     end
   end
 
+  desc "Import all NYC schools"
+  # task import_all_nyc_schools: :environment do
+  task import_all_nyc_schools: :environment do
+    csv_text = File.read('data/2016_-_2017_School_Locations.csv')
```

Comment:
I'm not 100% sure what's being done here, but it looks like maybe this is a script for ingesting a schools csv file.  If so, I'd like to specify the name of the csv file to ingest in the command line arguments, when you call the script.  If the script is called with wrong arguments, it should output a usage notification message.  We can then later call the script with on-demand arguments from a user-facing webpage.

I'd also like to make a short mock data file, with just a couple of schools in it, for the unit tests.